v22.1/oid.md

```diff
@@ -0,0 +1,29 @@
+---
+title: OID
+summary: The OID data type stores an unsigned 32 bit integer value.
+toc: true
+docs_area: reference.sql
+---
+
+The `OID` (Object Identifier) [data type](data-types.html) stores an unsigned 32 bit integer value.
```

Comment:
```suggestion
The `OID` (Object Identifier) [data type](data-types.html) stores an unsigned 32-bit integer value.
```

Reply:
Does "Object Identifier" need to be capitalized? Postgres docs imply no: https://www.postgresql.org/docs/current/datatype-oid.html